src/lib/dataEntities/networks.ts

```diff
@@ -284,6 +288,7 @@ export const l2Networks: L2Networks = {
       l2WethGateway: '0x7626841cB6113412F9c88D3ADC720C9FAC88D9eD',
     },
     nitroGenesisBlock: 0,
+    firstNitroBlock: 0,
```

Comment:
Should this be provided? Otherwise the assumption is only nitro messages are supported.

Reply:
Nova was only deployed with nitro, didnt exist in classic times